bugbug/models/regressionrange.py

```diff
@@ -80,6 +80,11 @@ def get_labels(self):
                     classes[bug_id] = 1
                 elif bug_data["cf_has_regression_range"] == "no":
                     classes[bug_id] = 0
+            # Check the regressed_by field
```

Comment:
Comments in general should describe "why" not "what" the code is doing. I do not think the comment here is necessary.

Reply:
thank you so much on the feed back i will give the update soon